src/pages/blog.jsx

```diff
@@ -1,7 +1,12 @@
 import React from "react";
+import styled from "styled-components";
+
 
 function Blog() {
-    return <div>Thoughts and Banter</div>;
+    const Blog = styled.div `
```

Comment:
You should put the styled component inside of `website/src/components/styled` since you made that folder specifically to hold styled components. Then you can import the `Blog` styled component from there to this file and use it.